e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -444,22 +448,62 @@ describe("scenarios > dashboard > tabs", () => {
       );
     });
 
+    const firstQuestion = () => {
+      return cy.request("GET", `/api/card/${ORDERS_QUESTION_ID}`).its("body");
+    };
+    const secondQuestion = () => {
+      return cy
+        .request("GET", `/api/card/${ORDERS_COUNT_QUESTION_ID}`)
+        .its("body");
+    };
+
+    firstQuestion().then(r => {
+      expect(r.view_count).to.equal(1);
+    });
+    secondQuestion().then(r => {
+      expect(r.view_count).to.equal(1);
+    });
+
     // Visit first tab and confirm only first card was queried
     visitDashboard(ORDERS_DASHBOARD_ID);
 
     cy.get("@firstTabQuery").should("have.been.calledOnce");
     cy.get("@secondTabQuery").should("not.have.been.called");
 
-    // Visit second tab and confirm only second card was queried
+    cy.wait(500); // this is needed to avoid flakes
```

Comment:
Did it flake locally or on CI here? It seems that requests have been resolved so `view_count` should've been updated already. Is there some delay after which BE updates it?